x-pack/plugin/sql/qa/server/src/main/resources/logs.csv

```diff
@@ -99,3 +99,4 @@ id,@timestamp,bytes_in,bytes_out,client_ip,client_port,dest_ip,status
 98,2017-11-10T21:12:24Z,74,90,10.0.0.134,57203,172.20.10.1,OK
 99,2017-11-10T21:17:37Z,39,512,10.0.0.128,29333,,OK
 100,2017-11-10T03:21:36Z,64,183,10.0.0.129,4541,172.16.1.1,OK
+101,2017-11-10T23:22:36Z,,,10.0.2.129,4541,172.20.11.1,OK
```

Comment:
Why was this added and what would have happened without it?

Reply:
@costin this a follow up to my suggestion [here](https://github.com/elastic/elasticsearch/pull/65796#discussion_r534770344) to allow a more complex test query to be added.